manifests/service.pp

```diff
@@ -39,8 +39,7 @@
       ensure     => running,
       hasstatus  => true,
       hasrestart => true,
-      enable     => true,
-      require    => Wso2base::Push_files[$service_refresh_file_list]
```

Comment:
What would be the reason for removing this line?
require    => Wso2base::Push_files[$service_refresh_file_list]

Reply:
Execution order will not differ. However, before executing Wso2base::Push_files[$service_refresh_file_list], we need to verify if $service_refresh_file_list is undefined or empty. This cannot be done using a require. Hence the require param is removed